RELEASES.md

```diff
@@ -380,3 +380,22 @@ The deprecated features are shown in the following table:
 | Built-in `aufs` snapshotter                                                      | containerd v1.5     | containerd v2.0            | Use `overlayfs` snapshotter       |
 | `cri-containerd-*.tar.gz` release bundles                                        | containerd v1.6     | containerd v2.0            | Use `containerd-*.tar.gz` bundles |
 | Pulling Schema 1 images (`application/vnd.docker.distribution.manifest.v1+json`) | containerd v1.7     | containerd v2.0            | Use Schema 2 or OCI images        |
+| CRI `v1alpha2`                                                                   | containerd v1.7     | containerd v2.0            | Use CRI `v1`                      |
+
+## Experimental features
+
+Experimental features are new features added to containerd which do not have the
+same stability guarantees as the rest of containerd. An effort is made to avoid
+breaking interfaces between versions, but changes to experimental features before
+being fully supported is possible. Users can still expect experimental features
+to be high quality and are encouraged to use new features to help them stabilize
+more quickly.
+
+| Component                                                                              | Initial Release | Target Supported Release |
```

Comment:
do we need to mark userns feature as experimental one?

Support for user namespaces in stateless pods ([KEP-127](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/127-user-namespaces)) (experimental) (https://github.com/containerd/containerd/pull/7679)

Reply:
We can always add later, I'm not sure

Reply:
I think we should add the #7679 as experimental feature since we already mark it as experimental in release note. :)

Reply:
added